easybuild/easyconfigs/c/Charm++/Charm++-6.8.0-beta2-changa-CrayIntel-2016.11.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CmdCp'
+
+name = 'Charm++'
+version = '6.8.0-beta2'
+versionsuffix = '-changa'
+
+homepage = 'https://charm.cs.illinois.edu/software'
+description = """Charm++ is a machine independent parallel programming system.
+Programs written using this system will run unchanged on MIMD machines with or
+without a shared memory."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2016.11'}
+toolchainopts = {'dynamic': False}
+
+# source_urls = ['http://charm.cs.illinois.edu/distrib/']
+sources = ['charm-%(version)s.tar.gz']
```

Comment:
if you are not using the downloaded version, use the absolute path here. That would avoid copying these sources to the private jenscscs sources folder.